### PR TITLE
FEATURE: adds last day to about page stats

### PR DESCRIPTION
**Pull request by jjaffeux**
Context of this change: https://meta.discourse.org/t/how-to-get-the-dau-and-mau-data-separately/186021

### CODE REVIEW

**Comment by jjaffeux** [comment]
Capture d’écran 2021-04-09 à 12 15 43
Capture d’écran 2021-04-09 à 12 16 00


Note that after these screens I also fixed a bug with user count not using real for 7/30 days:

Capture d’écran 2021-04-09 à 12 22 04

Which would cause difference in numbers (mostly visible for recent communities).

**Comment by riking** [comment]
The problem with this stat is that last day DAU is too volatile for weekends, which is why we only showed 7 day stats before. DAU actually does make sense as a admin report.

**Comment by jjaffeux** [comment]
It's certainly not a perfect number, I still think it's interesting to know the activity of the last day. eg: check if a big event has been creating a lot of activity in the last 24 hours.

I can also do the report thing, would be easy to add.

**Comment by pmusaraj** [comment]
I think users will find the "last day" label confusing. Does it mean "last 24 hours" or does it mean "yesterday" (i.e. midnight to midnight yesterday on the site's timezone)?

**Comment by jjaffeux** [comment]
> I think users will find the "last day" label confusing. Does it mean "last 24 hours" or does it mean "yesterday" (i.e. midnight to midnight yesterday on the site's timezone)?

Agreed, made the change in the locale

File: config/locales/client.en.yml [hunk 1]
```diff
@@ -282,6 +282,7 @@ en:
       moderators: "Moderators"
       stat:
         all_time: "All Time"
+        last_day: "Last 24 hours"
         last_7_days: "Last 7"
```
Comment (pmusaraj) [inline config/locales/client.en.yml:286]: We should add days to "Last 7" and "Last 30", now.

**Review by pmusaraj — APPROVED** [review]

File: config/locales/client.en.yml (hunk 1 above)
Comment (jjaffeux) [reply config/locales/client.en.yml]: yes sure :+1:

**Pull request merged**